File: solutions/java/armstrong-numbers/1/src/main/java/ArmstrongNumbers.java
class ArmstrongNumbers {

    boolean isArmstrongNumber(int numberToCheck) {
        char[] digits=Integer.toString(numberToCheck).toCharArray();
        int number=0;
        for (char i : digits){
            number+= Math.pow(Integer.parseInt((String.valueOf(i))), digits.length);
        }
        return (number==numberToCheck) ? true : false;

    }

}

File: solutions/python/armstrong-numbers/1/armstrong_numbers.py
def is_armstrong_number(number):
    number_str= str(number)
    armstrong = 0
    for i in range(len(number_str)):
        armstrong = armstrong + int(number_str[i])**len(number_str)
    if(armstrong==number):
        return True
    else:
        return False